services/rest_gate/cfg/config.yml

```diff
@@ -0,0 +1,7 @@
+# The IP and port to listen on.
+listen-address: 0.0.0.0:8090
```

Comment:
It seems here we should use 
```yml
server:
  listen-address: 0.0.0.0:8090
```
and drop the `REST_GW_SERVER_LISTEN_ADDRESS` from `rest_gate/docker-compose.yml`

Reply:
yeah, inaccurate rebase and bad copy/paste, fixed